CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [4.0.2] - 2022-08-19
```

Comment:
Also specify that you've added a new optional dependency and what the extra is called (functions).